pyproject.toml

```diff
@@ -17,7 +17,6 @@ flask-sqlalchemy = "^3.1.1"
 flask-wtf = "^1.2.1"
 gnupg = "^2.3.1"
 passlib = "^1.7.4"
-pymysql = "^1.1.0"
```

Comment:
The lockfile needs to be updated too as I'm fairly sure doing so will yank the `pymysql` deps and a few others